typescript/sdk/src/deploy/HyperlaneAppChecker.ts

```diff
@@ -122,15 +122,25 @@ export abstract class HyperlaneAppChecker<
     }
   }
 
-  async checkOwnership(
-    chain: ChainName,
-    owner: types.Address,
-    ownables: Ownable[],
-  ): Promise<void> {
+  async checkOwnership(chain: ChainName, owner: types.Address): Promise<void> {
+    const contracts = this.app.getContracts(chain);
+    const isOwnable = (contract: any): contract is Ownable => {
+      return (
+        contract !== null &&
+        typeof contract === 'object' &&
+        'owner' in contract &&
+        'transferOwnership' in contract
```

Comment:
the `in` keyword has some tricky pitfalls. Recommend something simpler for checks like this:
`contract.owner && contract.transferOwnership`